src/DataStructures/DataBox/DataBox.hpp

```diff
@@ -585,14 +585,12 @@ class DataBox<TagsList<Tags...>> {
   using compute_item_tags = databox_detail::dependent_items<
       tmpl::filter<tags_list, db::is_compute_item<tmpl::_1>>>;
 
-  /// A list of all the simple items, including subitems from the compute items
-  using item_tags = tmpl::list_difference<tags_list, compute_item_tags>;
+  /// A list of all the simple items, including subitems from the simple items
+  using simple_item_tags = tmpl::list_difference<tags_list, compute_item_tags>;
 
-  /// A list of all the simple items that have subitems, including the subitems
-  using subitem_expanded_tags = tmpl::flatten<tmpl::transform<
-      tmpl::filter<item_tags,
-                   tmpl::bind<databox_detail::has_subitems, tmpl::_1>>,
-      db::Subitems<tmpl::_1>>>;
+  /// A list of the subitems of the simple items
+  using subitem_expanded_tags =
```

Comment:
could this just be subitem_tags?

Reply:
Woops, I missed updating this. Maybe `simple_subitems_tags`?